Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Cultist Clothes

## Changelog:
- Scourge characters wear robes and hoods. Councilors and high nobles wear horned hoods. Their knights wear dark-colored armor.
- Court chaplains of Death religious family wear robes and horned hoods.
- Kel'Thuzad and other humans converted to Mord'truari have Scourge culture in the year 603 and 605 bookmarks.
- Added new clothes and headgears to the Barbershop.
- Scourge's **_Offer Vassalage_** interaction changes the culture of vassal unless it's vrykul or nerub.

# Tasks:
- [x] Localize draenei clothes
- [x] Add headgear and clothes to options
- [x] Scourge armor
- [x] Scourge vassals accept Scourge culture


## Tests:


# How to test:
Run the 603 and 605 bookmarks, check how Scourge characters look.

Resolved #425.
